components/ProfilePage/UserPageHeader.js

```diff
@@ -217,18 +307,26 @@ exported.fragments = {
       bio
       level
       appId
+
+      email
+      facebookId
+      githubId
+      availableAvatarTypes
+
       points {
         total
         nextLevel
       }
       ...AvatarData
       ...LevelProgressBarData
       ...EditProfileDialogUserData
+      ...EditAvatarDialogUserData
```

Comment:
Given the fact that
- All data in this fields are only required when EditAvatarDialog is opened; they are not required to render a profile page
- SSR for these fields won't work

I think it makes for sense if we perform a separate `useQuery` right inside render function of `EditProfileDialog` (i.e. after  `EditProfileDialog` is opened ) instead of putting its fragment in `UserPageHeader`'s `useQuery`. Plus, we could also drop `loadSelfOnlyFieldsForUser` lazy query & `useEffect` in `ProfilePage` if we load this fragment in `EditProfileDialog`.

(Seems that I should the similar to `EditProfileDialogUserData` as well; maybe I will refactor that in the future)